backend/src/index.js

```diff
@@ -94,6 +104,96 @@ async function main() {
   const wamp = setupWamp();
   console.log("Starting WAMP worker...");
   wamp.open();
+
+  const wampPublish = (topic, args) => {
+    const uri = `com.nearprotocol.${wampNearNetworkName}.explorer.${topic}`;
+    wamp.session.publish(uri, args);
+  };
+
+  const wampCall = async (args) => {
+    const uri = `com.nearprotocol.${wampNearNetworkName}.explorer.select`;
+    const res = await wamp.session.call(uri, args);
+    return res[0];
+  };
+
+  const regularQueryRPC = async () => {
+    try {
+      const finalTimestamp = await queryFinalTimestamp();
+      wampPublish("finalTimestamp", [finalTimestamp]);
+    } catch (error) {
+      console.warn("Regular querying RPC crashed due to:", error);
+    }
+    setTimeout(regularQueryRPC, regularQueryRPCInterval);
+  };
+  setTimeout(regularQueryRPC, 0);
+
+  const totalStats = async () => {
+    const [
+      totalBlocks,
+      totalTransactions,
+      totalAccounts,
+      lastDayTxCount,
+    ] = await Promise.all([
+      wampCall([`SELECT COUNT(*) as total FROM blocks`]),
+      wampCall([`SELECT COUNT(*) as total FROM transactions`]),
+      wampCall([`SELECT COUNT(*) as total FROM accounts`]),
+      wampCall([
+        `SELECT COUNT(*) as total FROM transactions
+        WHERE block_timestamp > (strftime('%s','now') - 60 * 60 * 24) * 1000`,
+      ]),
+    ]);
+    return [
+      totalAccounts.total,
+      totalBlocks.total,
+      totalTransactions.total,
+      lastDayTxCount.total,
+    ];
```

Comment:
Whenever you return more than a single value, let's prefer named values as it is very easy to mess things up in JS and left it unnoticed for quite some time.

```suggestion
    return {
      totalAccounts: totalAccounts.total,
      totalBlocks: totalBlocks.total,
      totalTransactions: totalTransactions.total,
      lastDayTxCount: lastDayTxCount.total,
    };
```